ext/mbstring/libmbfl/filters/mbfilter_utf8_mobile.c

```diff
@@ -47,6 +47,66 @@ static void mb_wchar_to_utf8_kddi_b(uint32_t *in, size_t len, mb_convert_buf *bu
 static size_t mb_utf8_sb_to_wchar(unsigned char **in, size_t *in_len, uint32_t *buf, size_t bufsize, unsigned int *state);
 static void mb_wchar_to_utf8_sb(uint32_t *in, size_t len, mb_convert_buf *buf, bool end);
 
+static int mbfilter_conv_map_tbl(int c, int *w, const unsigned short map[][3], int n)
```

Comment:
Seems like the return value could be ``bool``? Or is there a reason for it to be ``int``?

Reply:
No good reason... I just moved the existing, legacy code over as it was.

Will convert to `bool` now.